Analysis Project Rough Draft

are17 opened the pull request:
This is just a rough draft, please let me know what I can do to make this better. Thank You!

are17 committed:
Analysis Project Rough Draft

This is just a rough draft, please let me know what I can do to make this better. Thank You!

huoyanzhizhou commented:
Some basic things to check:
1. Your analysis project should be saved as .md file. 
2. Use more Markdown languages, see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
3. publish your .twb file on Tableau Public, and attach that in your .md file. See: https://dig-eg-gaz.github.io/how-to/serial-question-instructions/

are17 committed:
Add files via upload

I think this is saved as an .md file now and i think i have embedded the tableau correctly, I also attached my word art creation
please let me know if you need anything else. thank you!
